tests/jobs/test_backfill_job.py

```diff
@@ -1599,7 +1615,7 @@ def test_mapped_dag(self, dag_id, executor_name, session):
         self.dagbag.process_file(str(TEST_DAGS_FOLDER / f'{dag_id}.py'))
         dag = self.dagbag.get_dag(dag_id)
 
-        when = datetime.datetime(2022, 1, 1)
+        when = timezone.datetime(2022, 1, 1)
```

Comment:
Note: DB errors are thrown if a non-timezoned date is used, which makes me think these long running tests aren't being run in our CI? Since they would be failing. @potiuk do you have any context around that?

Reply:
Yep. Long running tests are NOT run on our CI at all :).

There is no more context than that :D

Reply:
:laughing: I meant more context for _why_ that's the case. I assume it's because folks just don't want to slow down the regular builds, but maybe there were timeouts on the test runners? Either way, what do you think about a scheduled CI run (once daily maybe?) on `main` which includes long running tests so that we at least get some coverage for them, thoughts?